server/gameconnection.py

```diff
@@ -547,6 +571,19 @@ async def handle_game_full(self):
         """
         pass
 
+    async def handle_established_peers(self, peerId: str, peerConnectedTo: str):
+        """
+        Send by the lobby when a peer connected to another peer. Message can be send multiple times, by multiple connections.
+
+        # Params
+        - peerId: The identifier of the peer that this connection received the message from
+        - peerConneectedTo: The list of identifiers that the given peer established a connection to. Is a list stored in a string, separated by spaces. As an example: "1321 22 33 43221 2132"
+        """
+
+        # peerConnectedToList = peerConnectedTo.split(" ")
```

Comment:
You can remove the commented out code. I think I’ll be able to figure that one out :)
```suggestion
```